VERSION

```diff
@@ -1 +1 @@
-latest
\ No newline at end of file
+v0.7.0
```

Comment:
v0.7.0-rc.0

Reply:
![image](https://user-images.githubusercontent.com/3085804/184080672-b1028093-19bc-4421-9ba2-57b1cc4bd31d.png)

Reply:
That's incorrect